### ISSUE TITLE
SOcial Media links are not accessible

### ISSUE THREAD

**Issue by zersiax** [budparr/gohugo-theme-ananke#145]
Hi,

On your theme's description page it lists that you aim to be accessible. However, the links for social media do not have textualcontent, making screen readers unable to determine where they go. ARIA-describeby is used to indicate the link opens in a new window, but the actual location of the link is unknown.
I have fixed it for now by adding text to the a tags in social-follow.html, but am fully blind myself so I have no idea to what degree that borks up the design. Can this be looked at?

**Comment by zersiax** [comment]
I would keep the title of opens in a new window, and use aria-label for the actual function of the link. I think that will work best. If not, you might need to aria-hidden=true the icon itself, but I think just aria-label should suffice. Shoot me a test version and I'll compare.
